.github/integration/scripts/charts/values.yaml

```diff
@@ -39,6 +39,7 @@ global:
     secretName: c4gh
     keyFile: c4gh.sec.pem
     publicFile: c4gh.pub.pem
+    c4ghPubKeyFile: c4gh.pub.pem
```

Comment:
I didn't think properly before, do we really need a new value here given that we have `publicFile`? I also don't see it being implemented in the deployment template (anything passing the value to the actual service).

Reply:
Right, thanks for the input. I'm pushing now some changes. publicFile should be used instead, changed all additions of c4ghpubkeyfile to publicfile and added in the auth deployment

Reply:
It should be now?
```
c4gh:
    secretName: c4gh
    keyFile: c4gh.sec.pem
    publicFile: c4gh.pub.pem
```

Reply:
Now the test file updates seem completely gone from the PR?

Reply:
I was not sure if `publicfile` was left out of config_test for reasons, so I thought best to not make changes there since I haven't added a separate field after all.

Reply:
Not sure if I'm missing something, but I'd expect some tests for the new functions/endpoint (which I think you had earlier, but I haven't done the work of going back and check).

Reply:
I added back the field `publicfile` in config_test and created a test for info `info_test.go`